web/src/components/lineage/Lineage.tsx

```diff
@@ -129,6 +131,48 @@ class Lineage extends React.Component<LineageProps, LineageState> {
     })
   }
 
+  getEdges = () => {
+    const selectedPaths = this.getSelectedPaths()
+
+    return g?.edges().map(e => {
+      const isSelected = selectedPaths.some((r: any) => e.v === r[0] && e.w === r[1])
+      return Object.assign(g.edge(e), { isSelected: isSelected })
+    })
+  }
+
+  getSelectedPaths = () => {
+    const paths = [] as Array<[MqNode, MqNode]>
+
+    const getSuccessors = (node: any) => {
```

Comment:
I think we can have a better typedef than any, right?

Reply:
Right, updated already, thanks